coldfront/core/project/views.py

```diff
@@ -3211,6 +3211,9 @@ def form_valid(self, form):
         try:
             project = self.__handle_create_new_project(form.cleaned_data)
             # Store form data in a request.
+
+            # changed to username, also changed VECTOR_PI_USERNAME to
```

Comment:
This comment doesn't seem to be necessary; it's already filtering on `username`.

(The value of `VECTOR_PI_USERNAME` did change recently, and I needed to make this change in my settings as well.)

Reply:
the problem was that it was filtering on "user" which didn't work, but I removed the comment anyway